op-challenger/scripts/list_games.sh

```diff
@@ -15,11 +15,9 @@ fi
 for i in $(seq 0 "${COUNT}")
 do
   GAME=$(cast call --rpc-url "${RPC}" "${FACTORY_ADDR}" 'gameAtIndex(uint256) returns(uint8, uint64, address)' "${i}")
-
-  SAVEIFS=$IFS   # Save current IFS (Internal Field Separator)
-  IFS=$'\n'      # Change IFS to newline char
-  GAME=($GAME) # split the string into an array by the same name
-  IFS=$SAVEIFS   # Restore original IFS
+  # Use read -ra to safely split the string into an array named GAME, assuming
+  # data is newline-separated.
+  IFS=$'\n' read -ra GAME <<< "$GAME"
```

Comment:
`-a` is not a standard read option so we need to stick to modifying IFS.

Reply:
Same as https://github.com/ethereum-optimism/optimism/pull/8594#discussion_r1424780088